src/main/java/io/antmedia/datastore/db/MapDBStore.java

```diff
@@ -355,19 +355,38 @@ public boolean delete(String id) {
 		}
 		return result;
 	}
+	@Override
+	public List<ConferenceRoom> getConferenceRoomList(int offset, int size, String sortBy, String orderBy, String search){
+		ArrayList<ConferenceRoom> list = new ArrayList<>();
+		synchronized (this) {
+			Collection<String> conferenceRooms = conferenceRoomMap.getValues();
```

Comment:
Protect `getConferenceRoom` the same way in `getVodList` . We need to check db consistency